components/Icons/PencilIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default function PencilIcon() {
```

Comment:
If we are using hero icons should we use the react library so that we can use its tailwind features?